Gems/AtomLyIntegration/AtomViewportDisplayInfo/Code/Source/AtomViewportDisplayInfoSystemComponent.cpp

```diff
@@ -220,8 +220,11 @@ namespace AZ::Render
             viewportContext->GetViewportSize().m_height,
             multisampleState.m_samples > 1 ? AZStd::string::format("MSAA %dx", multisampleState.m_samples).c_str() : "NoMSAA"
         ));
-                
-        DrawLine(AZStd::string::format("Render pipeline: %s", viewportContext->GetCurrentPipeline()->GetId().GetCStr()));
+
+        if(viewportContext->GetCurrentPipeline())   // avoid VR crash on nullptr
```

Comment:
Nit: Space between `if` and `(` for coding standards.
```suggestion
        if (viewportContext->GetCurrentPipeline())   // avoid VR crash on nullptr
```